tests/QuikGraph.Serialization.Tests/XmlSerializationTests.cs

```diff
@@ -98,14 +98,13 @@ private static IEnumerable<TestCaseData> XmlSerializationGraphTestCases
             {
                 yield return new TestCaseData(new AdjacencyGraph<Person, TaggedEdge<Person, string>>());
                 yield return new TestCaseData(new BidirectionalGraph<Person, TaggedEdge<Person, string>>());
+                yield return new TestCaseData(new UndirectedGraph<Person, TaggedEdge<Person, string>>());
             }
         }
 
         [TestCaseSource(nameof(XmlSerializationGraphTestCases))]
         public void SerializeToXml<TGraph>([NotNull] TGraph graph)
-            where TGraph 
-            : IVertexAndEdgeListGraph<Person, TaggedEdge<Person, string>>
-            , IMutableVertexAndEdgeSet<Person, TaggedEdge<Person, string>>
+            where TGraph: IMutableVertexAndEdgeSet<Person, TaggedEdge<Person, string>>
```

Comment:
Not sure if this was the best way to make it possible to also test serialization of the undirected graph, but it was the smallest change to make that made it possible to test.

Reply:
Well that's ok for me ;-)
We also can create a separate unit test for undirected graph but I think there this is enough, like for other graph types.